### PR TITLE
Electronize

### PR DESCRIPTION
**Pull request by rrreeeyyy**
I want to write the blog entry in local application. 
So I create `electron` entry-point script.

And fix some of codes to work with electron (Mainly, resource path problems).

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -6,24 +6,24 @@
 
   <!-- JS -->
   <script src="libs/js/mousetrap.min.js"></script>
-  <script src="//cdnjs.cloudflare.com/ajax/libs/ace/1.2.0/ace.js"></script>
+  <script src="https:////cdnjs.cloudflare.com/ajax/libs/ace/1.2.0/ace.js"></script>
```
Comment (sawa-zen) [inline public/index.html]: @rrreeeyyy 
スラッシュ多いように思うけどこれはOKなん？

Comment (rrreeeyyy) [reply public/index.html]: NO

Comment (sawa-zen) [reply public/index.html]: よろしゅう

**Comment by rrreeeyyy** [comment]
I just fixed wrong source URI. Could you merge it? @sawa-zen

**Comment by sawa-zen** [comment]
🙆‍♂️

**Pull request merged**